src/hooks/transactions/useGetTransactionLink.ts

```diff
@@ -0,0 +1,15 @@
+import { getTransactionLink } from 'utils';
+import { useCallback } from 'react';
+import { useGetNetworkConfig } from '../useGetNetworkConfig';
+
+export const useGetTransactionLink = () => {
```

Comment:
I don't think we need a hook for this. You can just create a simple function that accepts a address and an explorer address which is optional, and if the explorer address is not passed, extract it with explorerAddressSelector(store.getState())